apps/elf-council-frontend/src/ui/proposals/ProposalsPage.tsx

```diff
@@ -175,7 +173,7 @@ export default function ProposalsPage({
       account={account}
       signer={signer}
       proposal={selectedProposal}
-      unverified={!selectedProposal.createdTimestamp}
+      unverified={selectedProposal.snapshotId === "-1"}
```

Comment:
Then if we use empty string, this becomes just `unverified={!selectedProposal.snapshotId}`, which reads a little easier 😄